clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -9,9 +9,64 @@ import {
   fetchExperience,
 } from "fides-js";
 import { loadPrivacyCenterEnvironment } from "~/app/server-environment";
-import { lookupGeolocation, LOCATION_HEADERS } from "~/common/geolocation";
+import { LOCATION_HEADERS, lookupGeolocation } from "~/common/geolocation";
 
-const FIDES_JS_MAX_AGE_SECONDS = 60 * 60; // one hour
+// one hour, how long the client should cache fides.js for
+const FIDES_JS_MAX_AGE_SECONDS = 60 * 60;
+// one hour, how long until the custom-fides.css is refreshed
+const CUSTOM_FIDES_CSS_TTL_MS = 3600 * 1000;
+
+let cachedCustomFidesCss: string = "";
+let lastFetched: number = 0;
+
+async function fetchCustomFidesCss(
```

Comment:
Styling nit, but I think this would be better if you could move it's definition below the main entrypoint for this file which is the API `handler` function itself?

Reply:
Moved function